src/effects/native/balanceeffect.cpp

```diff
@@ -1,7 +1,11 @@
 #include "balanceeffect.h"
 
-namespace {
+#include "util/defs.h"
 
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
> turn the gain for the right channel of and bring it back by the bypass freq

I don't understand what you mean. Step-by-step, what did you do?

> I can lower the value. To which value?

I don't know. I'll experiment with that.